js/admin/pending-modals/pending-partner-modal.js

```diff
@@ -0,0 +1,76 @@
+window.showModal = function showModal(selectedModal, applicationId) {
+  const modal = new bootstrap.Modal(selectedModal);
+  const modalBody = document.querySelector('.partner-modal-body');
+
+  modal.show();
+  getPendingPartnerDetails(applicationId, modalBody);
+
+  console.log(applicationId);
+};
+
+window.dismissModal = function dismissModal(selectedModal) {
+  const modal = bootstrap.Modal.getInstance(selectedModal);
+  const form = selectedModal.querySelector('form');
+
+  resetFlags().then(stripInputListeners(form)).then(removeAllValidation(form));
+
+  modal.hide();
+};
+
```

Comment:
Same as above, move relevant AJAX to respective .js file.

The important thing to note here is that we want to separate the **logic** of the page from the **backend** that it needs. This is what we call separation of concerns.